File: Java Programs/Practice/PalindromeNumberGenerate.java
import java.util.Scanner;

public class PalindromeNumberGenerate
{
	public static void main(String args[])
	{
		Scanner sc = new Scanner(System.in);

		System.out.print("\n Enter a number : ");
		int n = sc.nextInt();
		
		System.out.print("The palindrome numbers are : ");

		int num = 0;
		int count = 0;

		while(count != 0)
		{
			int temp = num;
			
			result = 0;
			
			while(temp != 0)
			{
				int rem = result % 10;
				result = result * 10 + rem;
				result = result / 10;
			}
			
			if(result == num)
			{
				System.out.print(num + " ");
				count++;
			}
			
			num++;
		}
		
		System.out.println();
		
		sc.close();
	}
}	

File: Python Programs/palindrome_number_generate.py
n = int(input("Enter the no. of terms : "))

print("The palindrome numbers are :", end = " ")

num = 0
count = 0

while(count != n):
    temp = num

    result = 0

    while(temp != 0):
        rem = temp % 10
        result = result * 10 + rem
        temp = temp // 10

    if(result == num):
        print(num, end = " ")
        count += 1

    num += 1
